src/pzt/istio/sections/10-try/10-certificate.mdx

```diff
@@ -0,0 +1,39 @@
+<p>
+  To have Istio mesh running TLS on a Kubernetes cluster, Envoy sidecars will
+  need to be able to request a certificate issued by a trusted certificate
+  authority (CA). If you already have a private CA and root certificate, you can
+  skip to the automated renewal section below. If you need to create a CA, you
+  can:
+</p>
+<ul>
+  <li>
+    Use a{' '}
+    <a href="/signup?product=cm">
+      free hosted Smallstep Certificate Manager authority
+    </a>
+  </li>
+  <li>
+    Roll your own with{' '}
+    <a href="/docs/step-ca">
+      open source <inlineCode>step-ca</inlineCode>
+    </a>
+  </li>
+</ul>
+
+<Alert severity="warning">
+  This tutorial is designed for private networks, not publicly trusted websites
+  or endpoints. If you are looking to automate certificates for public websites,
+  servers, or endpoints, we recommend using{' '}
+  <a href="https://letsencrypt.org/">Let's Encrypt</a> and the ACME instructions
+  below.
+</Alert>
+
+Request a copy of your CA root certificate, which will be used to make
+sure each application can trust certificates presented by other
+applications.
+
+```shell
+step ca root ca.crt
+```
+
+Your certificate will be saved in `ca.crt`.
```

Comment:
This is a whole lot of duplication for a small copy difference. Maybe we should figure out how to create some smaller snippets that can be shared between docs in this template.